plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar38/python.jjt

```diff
@@ -943,7 +943,20 @@ void flow_stmt() #void: {}
 
 //return_stmt: 'return' [testlist]
 void return_stmt(): {}
-{ begin_return_stmt() [SmartTestList()] }
+{
```

Comment:
Please update the `//return_smt` comment above with the proper line from `grammar38`.